views/js/controller/list/index.js

```diff
@@ -52,12 +54,11 @@ define([
             .append($checkbox, $label);
     }
 
-    function addSquareBtn(title, icon, $listToolBar, position='rgt') {
+    function createButton(title, icon, position='rgt') {
```

Comment:
Is it a breaking change? If yes, maybe to avoid that we could:

- Deprecated the previous method.
- Delegate the call to the new method inside.
- Switch to the new method only here

Reply:
I've checked the code case, seems it is used only here :)

Reply:
Yes, it is a minor/private local (to the file) change. 
To make the action more specific. And slightly improves readability with new changes.